context.go

```diff
@@ -66,8 +66,9 @@ type Context struct {
 	// if this node never heard from validator i, LastSeenMessage[i] will be -1.
 	LastSeenMessage []*timer.HV
 
-	lastBlockTime  time.Time
-	lastBlockIndex uint32
+	lastBlockTimestamp uint64
+	lastBlockTime      time.Time
```

Comment:
Can we add some comment here, because the difference between `Time` and `Timestamp` is not obvious?